client/node_inspect.go

```diff
@@ -6,11 +6,15 @@ import (
 	"io/ioutil"
 
 	"github.com/docker/docker/api/types/swarm"
+	"github.com/pkg/errors"
 	"golang.org/x/net/context"
 )
 
 // NodeInspectWithRaw returns the node information.
 func (cli *Client) NodeInspectWithRaw(ctx context.Context, nodeID string) (swarm.Node, []byte, error) {
+	if nodeID == "" {
+		return swarm.Node{}, nil, errors.New("node id is required")
```

Comment:
We should be consistent here, how about an error like this: https://github.com/moby/moby/pull/34770/files#diff-18fbe225ba23b481ee707e859ff03b0cR22

and a test case

Reply:
I updated the error to be consistent with the volume inspect call and added a similar test case.